apps/bees/config.mk

```diff
@@ -60,7 +60,8 @@ CSRCS += \
 	$(APP_DIR)/src/ops/op_fade.c \
 	$(APP_DIR)/src/ops/op_gate.c \
 	$(APP_DIR)/src/ops/op_harry.c \
-	$(APP_DIR)/src/ops/op_hid_word.c \
+	$(APP_DIR)/src/ops/op_hid_word.c \	
```

Comment:
this tab breaks Make

Reply:
I have no clue about what is wrong here, but after copying the original version (and again I don't see any difference) it seems to work correctly for me. Therefore I replaced config.mk with the version I had in my computer.